forest_benchmarking/distance_measures.py

```diff
@@ -114,11 +114,68 @@ def hilbert_schmidt_ip(A, B):
     """
     return np.trace(np.matmul(np.transpose(np.conj(A)), B))
 
+def smith_fidelity(rho, sigma, power):
+    """
+    Computes the Smith fidelity F_S(rho,sigma,power) between two quantum states rho and sigma.
+
+    As the power is only defined for values less than 2, F_S > F.
+
+    :param rho: Is a D x D positive matrix.
+    :param sigma: Is a D x D positive matrix.
+    :param sigma: Is a positive scalar less than 2.
+    :return: Smith Fidelity which is a scalar.
+    """
+    if power <0:
+        raise ValueError("Power must be positive")
+    if power >= 2:
+        raise ValueError("Power must be less than 2")
+    return np.sqrt(fidelity(rho, sigma)) ** power
+
 
 # ============================================================================
 # Functions for quantum processes
 # ============================================================================
 
+def process_fidelity(pauli_lio0: np.ndarray, pauli_lio1: np.ndarray) -> float:
+    """Returns the fidelity between two channels, E and F, represented as Pauli Liouville matrix.
```

Comment:
```suggestion
    r"""Returns the fidelity between two channels, E and F, represented as Pauli Liouville matrix.
```

when your docstring has `\`'s in it, you have to tell python not to interpret them as escape codes